src/content/api/asset-transfers.mdx

```diff
@@ -115,6 +116,57 @@ Asset Transfer goes through different lifecycle stages.
 
 ---
 
+<Endpoint
+  path="/api/asset-transfer-requests"
+  filename="asset-transfers-request"
+>
+  ## Request an Asset Transfer <Badge type="experimental"/>
+
+
+  Request a transfer of an asset from a sender to the recipient.
+
+  <Properties>
+    <Property name="recipientAlias" type="string" required>
+      Phone number, email or handle of the person who request for asset transfer.
+    </Property>
+
+    <Property name="senderAccountId" type="string" required>
+      The id of the sender who receive the request.
+    </Property>
+
+    <Property name="assetType" type="string" required>
+      The type of asset being transferred. Currently, supports only `quartz.nzd` assets
+    </Property>
+
+    <Property name="description" type="string">
+      Shows up in transaction history against the transfer. 200 character limit.
+    </Property>
+
+    <Property name="message" type="string">
+      Shows up in transaction history against the transfer. 100 character limit.
+    </Property>
+
+    <Property name="senderName" type="string">
+      Human readable name for the sender. 30 character limit.
+    </Property>
```

Comment:
@Wulingoh Why are we providing senderName in the request endpoint? I thought this would only ever be defined by the Accepter.

Reply:
@bpal938, you are right. I will remove it.

Reply:
It would also need to be removed from the actual endpoint as well if it is there

Reply:
Have removed it from the endpoint .